ompi/mca/pml/ucx/pml_ucx_datatype.c

```diff
@@ -155,7 +157,7 @@ ucp_datatype_t mca_pml_ucx_init_datatype(ompi_datatype_t *datatype)
         (lb == 0))
     {
         ompi_datatype_type_size(datatype, &size);
-        PML_UCX_ASSERT(size > 0);
+        PML_UCX_ASSERT(size >= 0);
```

Comment:
size is `size_t`, so this assert does not make sense now - it will always be positive.